packages/integration-tests/suites/replay/requests/init.js

```diff
@@ -2,9 +2,9 @@ import * as Sentry from '@sentry/browser';
 
 window.Sentry = Sentry;
 window.Replay = new Sentry.Replay({
-  flushMinDelay: 500,
-  flushMaxDelay: 500,
-  useCompression: true,
+  flushMinDelay: 200,
```

Comment:
Did we change these on purpose? 🤔 IMHO it would be good to leave this at the same value to better replicate the regular "default" experience, wdyt?

Reply:
Ahh I think this is just a leftover from experimenting. Will change, thanks!

Reply:
Actually, I'll test this with another 150x run just to be sure 😅